.gitignore

```diff
@@ -294,5 +294,5 @@ __pycache__/
 # Functions Click
 functions-extensions/
 
-.vscode/
+
```

Comment:
I suspect these `.vscode/*` files were unintended additions to the PR :) . It happens to me all to the time, please look to remove them